subprojects/files/src/main/java/org/gradle/internal/file/FileMetadata.java

```diff
@@ -17,9 +17,9 @@
 package org.gradle.internal.file;
 
 /**
- * An immutable snapshot of the metadata of a file.
+ * The metadata of a file.
  */
-public interface FileMetadataSnapshot {
+public interface FileMetadata {
```

Comment:
The Javadoc for `AccessType` should not talk about "snapshots" anymore.

Reply:
I removed the mentions of `snapshot`.